config/toml.go

```diff
@@ -66,600 +68,6 @@ func WriteConfigFile(configFilePath string, config *Config) {
 
 // Note: any changes to the comments/variables/mapstructure
 // must be reflected in the appropriate struct in config/config.go.
-const defaultConfigTemplate = `# This is a TOML config file.
-# For more information, see https://github.com/toml-lang/toml
-
-# NOTE: Any path below can be absolute (e.g. "/var/myawesomeapp/data") or
-# relative to the home directory (e.g. "data"). The home directory is
-# "$HOME/.cometbft" by default, but could be changed via $CMTHOME env variable
-# or --home cmd flag.
-
-# The version of the CometBFT binary that created or
-# last modified the config file. Do not modify this.
-version = "{{ .BaseConfig.Version }}"
-
-#######################################################################
-###                   Main Base Config Options                      ###
-#######################################################################
-
-# TCP or UNIX socket address of the ABCI application,
-# or the name of an ABCI application compiled in with the CometBFT binary
-proxy_app = "{{ .BaseConfig.ProxyApp }}"
-
-# A custom human readable name for this node
-moniker = "{{ .BaseConfig.Moniker }}"
-
-# Database backend: goleveldb | cleveldb | boltdb | rocksdb | badgerdb | pebbledb
-# * goleveldb (github.com/syndtr/goleveldb)
-#   - UNMAINTAINED
-#   - stable
-#   - pure go
-# * cleveldb (uses levigo wrapper)
-#   - DEPRECATED
-#   - requires gcc
-#   - use cleveldb build tag (go build -tags cleveldb)
-# * boltdb (uses etcd's fork of bolt - github.com/etcd-io/bbolt)
-#   - DEPRECATED
-#   - EXPERIMENTAL
-#   - stable
-#   - use boltdb build tag (go build -tags boltdb)
```

Comment:
These lines, as well, have disappeared from the embedded file

Reply:
same as above